gmaildump/gmailhistory.py

```diff
@@ -217,6 +252,7 @@ def store_msgs_in_db(self, msgs_list):
 
```

Comment:
If this function is doing the work of storing the messages in db, what where the other two functions doing, little bit confusion for me.